data/mods/Magiclysm/npc/TALK_FORGE_HELEN_TAVREL.json

```diff
@@ -0,0 +1,112 @@
+      { "skill": "computer", "bonus": { "one_in": 2 } },
+      { "skill": "electronics", "bonus": { "rng": [ 0, 2 ] } },
+      { "skill": "speech", "bonus": { "rng": [ 1, 3 ] } },
+      { "skill": "barter", "bonus": { "rng": [ 3, 5 ] } }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "NC_FORGE_REAGENT_clothes",
+    "subtype": "collection",
+    "entries": [
+      { "item": "baldric_holster", "prob": 100, "contents-group": "pirate_loadout" },
+      { "item": "cavalry_sabre_plus_one", "prob": 100, "container-item": "scabbard" },
+      { "item": "bandana", "prob": 100 },
+      { "item": "mboots_escape", "prob": 100 },
+      { "item": "parabolan_enchanted_breeches", "prob": 100 },
+      { "item": "enchanted_pirate_blouse", "prob": 100 },
+      { "item": "flintlock_pouch", "contents-group": "flintlock_pouch_reenactor" }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "NC_FORGE_REAGENT_wield",
+    "subtype": "collection",
+    "items": [ { "item": "enchanted_blackpowder_pistol", "prob": 100, "charges": 1 } ]
+  },
+  {
+    "type": "item_group",
+    "subtype": "collection",
+    "id": "pirate_loadout",
+    "entries": [
+      { "item": "enchanted_blackpowder_pistol", "ammo-item": "flintlock_ammo", "charges": 1 },
+      { "item": "enchanted_blackpowder_pistol", "ammo-item": "flintlock_ammo", "charges": 1 },
+      { "item": "enchanted_blackpowder_pistol", "ammo-item": "flintlock_ammo", "charges": 1 },
+      { "item": "enchanted_blackpowder_pistol", "ammo-item": "flintlock_ammo", "charges": 1 },
+      { "item": "enchanted_blackpowder_pistol", "ammo-item": "flintlock_ammo", "charges": 1 },
+      { "item": "enchanted_blackpowder_pistol", "ammo-item": "flintlock_ammo", "charges": 1 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "subtype": "collection",
+    "id": "NC_FORGE_LORD_STORE_REAGENT",
+    "items": [
+      { "group": "organic_materials", "count": [ 13, 18 ], "prob": 100 },
+      { "group": "magic_shop_potions", "count": [ 1, 3 ], "prob": 100 },
+      { "group": "potions_common", "count": [ 5, 8 ], "prob": 50 },
+      { "group": "spellbook_loot_1", "prob": 75, "count": [ 1, 3 ] }
+    ]
+  },
+  {
+    "id": "TALK_FORGE_LORD_REAGENT",
+    "type": "talk_topic",
+    "dynamic_line": {
+      "u_is_wearing": "badge_marshal",
+      "yes": [
+        "Would you try to capture me?",
+        "I've killed many a government-sanctioned hunter in my life.",
+        "Aren't we all lucky that you are probably the last lawman alive in miles?"
+      ],
+      "no": [ "Welcome to the Forge of Wonders.", "Reagents and blackpowders!", "Ahoy and other nautical terms!" ]
+    },
+    "responses": [
+      { "text": "Let's trade.", "effect": "start_trade", "topic": "TALK_FORGE_LORD_REAGENT" },
+      { "text": "What are you doing here?", "topic": "TALK_FORGE_REAGENT_DOING" },
+      { "text": "Well, bye.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_FORGE_REAGENT_DOING",
```

Comment:
a "reagent" is a thing you use in potion-making. a "regent" is someone who has authority and oversees something. minor nit